test/modules/basic.js

```diff
@@ -223,49 +226,50 @@
         viewer.open('/test/data/testpattern.dzi');
     });
 
-    QUnit.test('FullScreen', function(assert) {
-        const done = assert.async();
-        if (!OpenSeadragon.supportsFullScreen) {
-            assert.expect(0);
-            done();
-            return;
-        }
-
-        viewer.addHandler('open', function () {
-            assert.ok(!OpenSeadragon.isFullScreen(), 'Started out not fullscreen');
-
-            const checkEnteringPreFullScreen = (event) => {
-                viewer.removeHandler('pre-full-screen', checkEnteringPreFullScreen);
-                assert.ok(event.fullScreen, 'Switching to fullscreen');
-                assert.ok(!OpenSeadragon.isFullScreen(), 'Not yet fullscreen');
-            };
-
-            const checkExitingFullScreen = (event) => {
-                viewer.removeHandler('full-screen', checkExitingFullScreen);
-                assert.ok(!event.fullScreen, 'Disabling fullscreen');
-                assert.ok(!OpenSeadragon.isFullScreen(), 'Fullscreen disabled');
-                done();
-            }
-
-            // The 'new' headless mode allows us to enter fullscreen, so verify
-            // that we see the correct values returned. We will then close out
-            // of fullscreen to check the same values when exiting.
-            const checkAcquiredFullScreen = (event) => {
-                viewer.removeHandler('full-screen', checkAcquiredFullScreen);
-                viewer.addHandler('full-screen', checkExitingFullScreen);
-                assert.ok(event.fullScreen, 'Acquired fullscreen');
-                assert.ok(OpenSeadragon.isFullScreen(), 'Fullscreen enabled');
-                viewer.setFullScreen(false);
-            };
-
-
-            viewer.addHandler('pre-full-screen', checkEnteringPreFullScreen);
-            viewer.addHandler('full-screen', checkAcquiredFullScreen);
-            viewer.setFullScreen(true);
-        });
-
-        viewer.open('/test/data/testpattern.dzi');
-    });
+    // TODO: can this be enabled without breaking tests due to lack of short-duration user interaction?
+    // QUnit.test('FullScreen', function(assert) {
```

Comment:
I forgot why we're disabling this here. Does this PR introduce something that breaks it?

Reply:
So it's unrelated to the WebGL change but it's part of this PR since it also needed fixing and was blocking you. That's fine... Just checking.